src/core/guards/__tests__/sensitive-operations.guard.spec.ts

```
@@ -0,0 +1,224 @@
+
+    it('should handle empty user ID', async () => {
+      mockRequest.user = { id: '' };
+
+      const result = await guard.canActivate(mockContext);
+      expect(result).toBe(true);
+    });
+  });
+
+  describe('environment variable configuration', () => {
+    afterEach(() => {
+      delete process.env.RATE_LIMIT_SENSITIVE_WINDOW_MS;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)